docs/releases/1.10.txt

```diff
@@ -209,7 +209,8 @@ Templates
 Tests
 ^^^^^
 
-* ...
+* Tests and test cases can be marked with tags and ``--tag`` and ``--exclude-tag``
```

Comment:
Please link to the new topic documentation and also document the `--tag/--exclude-tag` options under `.. django-admin:: test` in `docs/ref/django-admint.xt` (then you can link those too).
Please wrap docs at 79 characters.